src/contrib-scope.md

```diff
@@ -1,5 +1,8 @@
 # Scope
 
+The [`getrandom`](https://github.com/rust-random/getrandom/) library deals with
+platform-specific interfaces.
```

Comment:
I would mention that the interfaces are for providing entropy/randomness from the OS.